pkg/option/config.go

```diff
@@ -616,6 +616,9 @@ const (
 	// PolicyMapMax defines the maximum policy map limit.
 	PolicyMapMax = 1 << 16
 
+	// EgressPolicyMax defines the maximum egress policy map limit.
+	EgressPolicyMax = 1 << 14
```

Comment:
This was defined in the `maps/egress` package, can we keep it there?

Reply:
unless I missing something. this wasn't defined in the `maps/egress`. so we should keep it here. right?

Reply:
Apologies, I misspelled the package name, I meant this: https://github.com/cilium/cilium/blob/5449a9fd7e147d59eda65b3ee2a31c6fd7a2db77/pkg/maps/egressmap/egress.go#L9